openai/api_resources/engine.py

```diff
@@ -30,3 +30,6 @@ def generate(self, timeout=None, **params):
 
     def search(self, **params):
         return self.request("post", self.instance_url() + "/search", params)
+
+    def embeddings(self, **params):
```

Comment:
Can you take explicit parameters? I realize `**params` is common in this library but it's still bad.